docs/docs-content/vertex/system-management/enable-non-fips-settings/use-non-fips-addon-packs.md

```diff
@@ -12,79 +12,76 @@ tags: ["vertex", "non-fips"]
 
 
 Palette VerteX provides the following FIPS-compliant infrastructure components in Kubernetes clusters it deploys. Review [FIPS-Compliant Components](../../fips/fips-compliant-components.md) to learn more.
-
-<br />
-
     
 - Operating System (OS)
-
 - Kubernetes
-
 - Container Network Interface (CNI)
-
 - Container Storage Interface (CSI)
 
-As shown in the screenshot below, the FIPS-compliant icon used to indicate full FIPS compliance is displayed next to Palette VerteX infrastructure components in the cluster profile stack. To learn about other icons Palette VerteX applies, refer to [FIPS Status Icons](../../fips/fips-status-icons.md).
+VerteX provides the flexibility to utilize non-FIPS packs. To do this, you must add a non-FIPS compliant pack registry. Review the [Add a Registry](../add-registry.md) guide. 
```

Comment:
We should mention here that the user can choose between adding the registries at the tenant scope or at the system scope. Registries at the system scope are available to all tenants.